src/utils/isValidZipCode.js

```diff
@@ -0,0 +1,18 @@
+const ZIP_COUNT = 48073;
+
+/**
+ * Check if a given integer is a valid zip code (between 1000 and 48073)
+ * @param {Number} code zip code
+ * @returns {Boolean}
+ */
+const isValidZipCode = (code) => {
+  const parsedCode = parseInt(code, 10);
+  return (
+    !Number.isNaN(parsedCode) &&
```

Comment:
We can remove this, as if the number is `NaN` our `Number.isInteger(parsedCode)` will return false
```suggestion
```